CMakeLists.txt

```diff
@@ -563,7 +552,6 @@ endif()
 add_compile_options(-Dgoogle=__tc_google)
 add_compile_options(-DCoreML=__tc_CoreML)
 add_compile_options(-DHAVE_PTHREAD=1)
-add_compile_options(-DEIGEN_MPL2_ONLY)
```

Comment:
Do we not need this anymore? I thought we did...

Reply:
It was already defined above -- this deletes the duplicate.